framework/platform/ios/ios_window.mm

```diff
@@ -69,7 +69,7 @@
 
 float IosWindow::get_dpi_factor() const
 {
-    return [[UIScreen mainScreen] nativeScale] * (([[UIDevice currentDevice] userInterfaceIdiom] == UIUserInterfaceIdiomPad) ? 132 : 163);
+	return 1.0;
```

Comment:
This is almost nothing more than a question; Is this considered the "right way" to get the dpi for text for all iOS devices?  It seems that a 1-1 ratio might be wrong in some cases?  I guess not given the displays themselves are small and the pixel pitch is small.

Reply:
I think @billhollings and I found the same issue that was causing a UI Overlay scaling problems on iOS.  The way I solved it was the following.  This works for real iOS devices and the iOS Simulator:

In ViewController.mm:

```
-->	self.vulkan_view.contentScaleFactor = UIScreen.mainScreen.nativeScale;
        context = create_platform_context((int)args.count, const_cast<char**>(argv));
        _context = (vkb::IosPlatformContext*)context.get();
        _context->view = self.vulkan_view;
        _code = (vkb::ExitCode)platform_main(*context);
        uint32_t fps = 60;
```

In ios_window.mm

`return [[UIScreen mainScreen] nativeScale];`

Reply:
Looking at this again, I think the following is a bit more semantically correct:

In **ViewController.mm**:

```
...
-->	self.vulkan_view.contentScaleFactor = UIScreen.mainScreen.nativeScale;
        context = create_platform_context((int)args.count, const_cast<char**>(argv));
        _context = (vkb::IosPlatformContext*)context.get();
...
```

In **ios_window.mm**:

```
float IosWindow::get_dpi_factor() const
{
	return 1.0;
}

float IosWindow::get_content_scale_factor() const
{
	return [[UIScreen mainScreen] nativeScale];
}
```

I think @billhollings is correct to return 1.0 for `get_dpi_factor()`.  My changes for setting and getting the view's `contentScaleFactor` are also okay, but they offset each other for the imgui UI Overlay and the code would still work without adding this part.

Reply:
> I think @billhollings is correct to return 1.0 for `get_dpi_factor()`. My changes for setting and getting the view's `contentScaleFactor` are also okay, but they offset each other for the imgui UI Overlay and the code would still work without adding this part.

Okay. How about I leave it as it is now (returning `1.0`), and adding additional logic around screen attributes can be left for another PR?

It is currently displaying nicely on both iPhone and iPad. On an iPad, the app display Idiom seems to stay as iPhone, so it uses the standard iPad PiP when an app is only designed for iPhone. As a result, it looks the same on both iPhone and iPad.

Reply:
> How about I leave it as it is now (returning 1.0), and adding additional logic around screen attributes can be left for another PR?

That's fine for now, but the iOS view `contentScaleFactor` and `get_content_scale_factor()` need to be setup for other samples in the project (e.g. _swapchain_recreation_).  I can submit these changes after your PR is merged.

I will work on your review request tomorrow.  There's an additional error path that needs to be handled in `main_loop_frame()` when the sample is not properly specified (e.g. cmd line typo). Should be a simple fix.